backend/src/services/review.ts

```diff
@@ -89,6 +89,14 @@ export async function createAccessRequestReviews(model: ModelDoc, accessRequest:
   await Promise.all(createReviews)
 }
 
+export async function removeAccessRequestReview(accessRequestId: string) {
+  const accessRequestReview = await findReviewForAccessRequest(accessRequestId)
+
+  await accessRequestReview.delete()
+
+  return { accessRequestId }
```

Comment:
I would probably return the result of the call to delete the document in Mongo rather than the access request ID that was provided via the method parameter.